content/community/Twitter.md

```diff
@@ -0,0 +1,9 @@
+---
+          title: "FOLLOW US ON TWITTER"
+          link: https://twitter.com/hashtag/hpedevcom?src=hash
+          linkname: FOLLOW US ON TWITTER
```

Comment:
Should be sentence-case rather than uppercase according to figma

Reply:
fixed